.circleci/config.yml

```diff
@@ -65,48 +65,47 @@ workflows:
             tags:
               only: /^v.*/
 
-      # TODO: Uncomment it after we deployed helm 3 supported app-operator into all installations.
-      # - architect/push-to-app-collection:
-      #     context: architect
-      #     name: push-chart-operator-to-aws-app-collection
-      #     app_name: "chart-operator"
-      #     app_collection_repo: "aws-app-collection"
-      #     unique: "true"
-      #     requires:
-      #       - push-chart-operator-to-control-plane-app-catalog
-      #     filters:
-      #       branches:
-      #         ignore: /.*/
-      #       tags:
-      #         only: /^v.*/
+      - architect/push-to-app-collection:
```

Comment:
Can you either add the same comment as app-operator, or remove the comment? 

I don't care which. I just care that we are consistent. 😉  Thx!

Reply:
I will delete comments in app-operator, thx!